index.html

```diff
@@ -1,52 +1,62 @@
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+
   <title>Yousef Hesham</title>
 </head>
-<body>
-  <div class ="wrapper">
-<head class="head-toolbar">
-  <nav class ="navbar">
-    <ul class ="toolbar-items">
-      <li>
-        <a href="#"> Yousef Hesham </a>
-      </li>
-      <li>
-        <img src="assets/images/hamburger.svg" alt="">
-      </li>
-    </ul>
-  </nav>
-</head>
 
-<section class ="headline">
-  <h2 class="welcome-message">
-    Hey there.I`m Mariane <span class="job-title">
-      I`m a software developer
-    </span>
-  </h2>
-  <p class="description">
-    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-    don’t hestiate to contact me.
-  </p>
-  <div class="social">
-  <ul class="social-icons"> 
-    <li> 
-      <img src="assets/images/twitter.svg" alt="">
-      <img src="assets/images/Linkedin icon.svg" alt="">
-      <img src="assets/images/medium.svg" alt="">
-      <img src="assets/images/github.svg" alt="">
-      <img src="assets/images/angellist.svg" alt="">
+<body>
+  <div class="wrapper">
 
+    <header class="head-toolbar">
+      <nav class="navbar">
+        <ul class="toolbar-items">
+          <li>
+            <a class="toolbar-title" href="#">
+              <h1>Welcome</h1>
+            </a>
+          </li>
+          <li class="hamburger">
+            <a href="#"> <img src="assets/images/hamburger.svg" alt="" /> </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
 
-    </li>
-  </ul>
-</div>
-</section>
-</div>
-  
+    <section class="headline">
+      <h2 class="welcome-message">
+        Hey there.I`m Yousef <br>
+        <span> I`m a software developer </span>
+      </h2>
+      <p class="description">
+        I can help you build a product , feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="social">
+        <ul class="social-icons">
+          <li class="items">
+            <a href="#"> <img src="assets/images/twitter.svg" alt="" /></a>
+          </li>
+          <li class="items">
+            <a href="#"> <img src="assets/images/Linkedinicon.svg" alt="" /> </a>
+          </li>
+          <li class="items" > <a href="#"> <img src="assets/images/medium.svg" alt="" /> </a> </li>
+          <li class="items">
+           <a href="#"> <img src="assets/images/github.svg" alt="" /> </a>
+          </li>
+          <li class="items">
+            <a href="#"> <img src="assets/images/angellist.svg" alt="" /> </a>
+          </li>
+        </ul>
```

Comment:
- [ ] Please make sure the section has the margins (top, bottom, left, right) defined in the design

![r5](https://user-images.githubusercontent.com/63972670/158830136-5e909af3-7d1b-47a7-b461-c47ca61751ac.JPG)

![r4](https://user-images.githubusercontent.com/63972670/158830825-bcde0d63-bc40-4044-af1c-7b5df5225c94.JPG)